### DISCUSSION TITLE
Hiding Navigation in Slideshow widget

### DISCUSSION THREAD

**Discussion by Brandocracy** [templaza/astroid-framework#846] [Ideas]
The navigation icons in the Astroid Slideshow widget (back, forward, and bottom nav) are easy enough to hide with CSS when you want to autoplay only with no nav, or you have only one image. But a suggestion would be to be able to hide them in the widget settings.
![navicons](https://github.com/user-attachments/assets/a7aa8e54-2d07-452c-90d1-18a6ead10ac4)

**Comment by Banuk1616** [comment]
I use several slideshows on different pages of my website, which is currently under development, and I first tried using css. This had the disadvantage that the nav elements were hidden in all slideshows. I solved this by creating my own slideshow widget without nav elements from the Astroid Slideshow Widget. This is relatively easy and you are very flexible. Of course, it would be more convenient if you could hide the nav elements in the slideshow widget, that would save some work. I therefore like your idea.

Best Regard

**Comment by felixkat** [comment]
I've created two options for indicators and controls, I'll upload a pull request over the weekend but will provide a link for you to test first.

![image](https://github.com/user-attachments/assets/6861deca-8613-4116-82a0-f6c617176b80)

**Comment by felixkat** [comment]
@Banuk1616 This is in the latest code now.

**Reply by Brandocracy** [reply]
That looks great! Thanks!

**Comment by Banuk1616** [comment]
@flexikat 
This is a great option for the slideshow widget. Thank you very much for your work.
